lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -500,12 +515,12 @@ def trace_type(cls: _T) -> _T:
     It assumes that this class is only instantiated after a charm type decorated with `@trace_charm`
     has been instantiated.
     """
-    logger.info(f"instrumenting {cls}")
+    # logger.info(f"instrumenting {cls}")
```

Comment:
Prtetty useful for debugging, but maybe we want to remove the commented out lines before the merge?

Reply:
yeh I was considering to keep them there because they're very useful in development. should we add a separate logger for them perhaps, and manually enable that when needed?

Reply:
Sounds like a great idea :)